### DISCUSSION TITLE
Are you sure about the name "Servo"?

### DISCUSSION THREAD

**Discussion by mcallegari** [servo/servo#30619] [General]
Sorry to raise this, but seeing all the fuzz that went into the "master/slave" topic, I just wanted to inform you that in Italian the word "servo" is the English word "servant".
The "servo" root is the latin word "servus" which also means "slave" or "submissive".

I take you meant "servo" as used in electronics like "servomotor", but still someone could object the name.
Personally I don't care much (not even about dropping master/slave since they were clearly related to electronics) but if you care then this post might be useful.

As for the project, keep up the good work! Big fan of open source and all the good stuff that comes out of it!

**Comment by fabricedesre** [comment]
The name originally comes from https://en.wikipedia.org/wiki/Tom_Servo . Don't ask me why, I don't know more :)

**Comment by mrego** [comment]
BTW, the issue with *servo* in Italian is the same in Portuguese and Galician where it also means *servant* or *serf*.
